partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnConcat.kt

```diff
@@ -8,34 +8,45 @@ import org.partiql.spi.function.Parameter
 import org.partiql.spi.value.Datum
 import org.partiql.types.PType
 
-internal val Fn_CONCAT__STRING_STRING__STRING = Function.static(
-
+internal val Fn_CONCAT__CHAR_CHAR__CHAR = Function.static(
     name = "concat",
-    returns = PType.string(),
+    returns = PType.character(256), // TODO: Handle length
```

Comment:
Should these TODOs for length be handled in this PR or tracked somewhere else?

Reply:
The work for this has begun in #1647.